front-end/README.md

```diff
@@ -0,0 +1,26 @@
+# Function-Stream
+
+
+## Editor Config
+
+### IDEA / WebStorm
+
+install plugin `prettier` (may have been pre-installed in webstorm)
```

Comment:
Please use the format like `* Install plugin `prettier` (may have been pre-installed in webstorm)`